python/cuml/cuml/cluster/hdbscan/hdbscan.pyx

```diff
@@ -211,9 +213,9 @@ cdef class _HDBSCANState:
             n_rows,
             cluster_selection_method,
             deref(temp_buffer),
-            <bool> self.allow_single_cluster,
-            <int> self.max_cluster_size,
-            <float> self.cluster_selection_epsilon
+            <bool> model.allow_single_cluster,
+            <int> model.max_cluster_size,
+            <float> model.cluster_selection_epsilon
```

Comment:
Thinking about what kind of common test we could add to catch things like this. We'd need a test that constructs a "scikit-learn" instance with a non-default value for a hyper-parameter, then calls `Foo.from_sklearn` and checks that the non default value got passed on properly. Except when the hyper-parameter doesn't exist/make sense for the cuml model.

This means I am not sure how we'd write a common test. It would need to know specifics about each estimator it tests :-/